### DISCUSSION TITLE
Só vim elogiar! PERFEITO!

### DISCUSSION THREAD

**Discussion by vitorhugosg** [laradumps/laradumps#113] [General]
Caraca que massa isso que vocês fizeram, eu tava atraz de algo assim, eu não consegui colocar o xdebug, isso facilitou minha vida! parabéns a todos envolvidos!

**Comment by eness** [comment]
I'm new to the package. I want to thank also everyone who contributed!

**Comment by luanfreitasdev** [comment]
Muito obrigado pelo feedback!  ❤️.
